onnxruntime/core/session/inference_session.cc

```diff
@@ -935,9 +935,11 @@ Status InferenceSession::PartitionOrtFormatModel(onnxruntime::Graph& graph,
                                                        GraphPartitioner::Mode::kOrtFormatLoad,
                                                        &compiled_kernel_hashes));
 
+#if defined(ENABLE_ORT_FORMAT_LOAD)
```

Comment:
this changes the behavior of ORT regardless building wasm or not. is this expected?

Reply:
SetCompiledKernelHashes is declared inside this macro, so there is a compile error without it.

Reply:
got. will keep it